packages/jest-reporters/src/__tests__/__snapshots__/GithubActionsReporter.test.js.snap

```diff
@@ -0,0 +1,8 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`reporter extracts the correct filename, line, and column 1`] = `
+"::error file=/home/runner/work/jest/jest/some.test.js,line=4,col=17::%0A              Error: <dim>expect(</><red>received</><dim>).</>toBe<dim>(</><green>expected</><dim>) // Object.is equality</>%0A%0A              %0A%0A              Expected: <green>"b"</>%0A%0A              Received: <red>"a"</>%0A%0A                  at Object.<anonymous> (/home/runner/work/jest/jest/some.test.js:4:17)%0A%0A                  at Object.asyncJestTest (/home/runner/work/jest/jest/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:106:37)%0A%0A                  at /home/runner/work/jest/jest/node_modules/jest-jasmine2/build/queueRunner.js:45:12%0A%0A                  at new Promise (<anonymous>)%0A%0A                  at mapper (/home/runner/work/jest/jest/node_modules/jest-jasmine2/build/queueRunner.js:28:19)%0A%0A                  at /home/runner/work/jest/jest/node_modules/jest-jasmine2/build/queueRunner.js:75:41%0A%0A                  at processTicksAndRejections (internal/process/task_queues.js:93:5)%0A            
```

Comment:
should we strip the ansi escape codes?

Reply:
Yeah, probably 🤔 

I was under the impression for a while that GH's annotations supported them, but I might've misread [the docs that I found](https://github.com/actions/toolkit/blob/e76decaf8a52bc981d17ddd3efcca548f6bc1a0e/packages/core/README.md#styling-output) -- which might really just refer to action logs (which are different from annotations) 🤷‍♂️ 

Does Jest have a helper function that we could use here?

Reply:
no, just use `strip-ansi`

Reply:
1ae84fbcbfc8b863f1aaff55e73792c9915564cb